packages/synapse-react-client/.storybook/preview.jsx

```diff
@@ -108,8 +108,9 @@ initialize({
   onUnhandledRequest: 'bypass',
 })
 
+export const loaders = [mswLoader]
+
```

Comment:
Using the (new-ish) loader instead of the decorator should ensure the service worker is initialized before the story runs, eliminating race conditions.